README.md

```
@@ -1,299 +1,180 @@
-1. Install mise: [Getting started](https://mise.jdx.dev/getting-started.html)
-2. Run setup:
+</div>
 
-    ```bash
-    mise install        # Installs Python 3.12 and Java 11
-    mise run setup      # Installs dependencies and pre-commit hooks
-    ```
+# Deepnote Toolkit
 
-#### Option 2: Manual setup
+The Deepnote Toolkit is a Python package that powers the [Deepnote notebook environment](https://github.com/deepnote/deepnote/). It provides the essential functionality that runs in user workspaces, enabling interactive data science workflows with SQL, visualizations, and integrations. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)